File: peek_core_device/_private/tuples/StringCapToggleActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_core_device._private.PluginNames import deviceTuplePrefix


@addTupleType
class StringCapToggleActionTuple(TupleActionABC):
    __tupleType__ = deviceTuplePrefix + "StringCapToggleActionTuple"

    stringIntId = TupleField()

File: peek_core_device/plugin-module/_private/tuples/StringCapToggleActionTuple.ts
                                       import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {deviceTuplePrefix} from "../PluginNames";

@addTupleType
export class StringCapToggleActionTuple extends TupleActionABC {
    static readonly tupleName = deviceTuplePrefix + "StringCapToggleActionTuple";

    stringIntId: number;

    constructor() {
        super(StringCapToggleActionTuple.tupleName)
    }
}